draft-ietf-rats-corim.md

```diff
@@ -1756,43 +1782,38 @@ For example, if the Evidence format is known in advance, CoRIMs using a profile
+If CoBOMs are required by policy, the selected tags are considered *inactive*.
+An *inactive* tag MUST NOT be used during the Appraisal Procedure.
+If CoBOMs are not required by policy, the selected tags are considered *active* and thus MUST be used during the Appraisal Procedure.
+
 Later stages will further select the CoRIMs appropriate to the Evidence Appraisal stage.
 
 #### Tags Extraction and Validation
 
-The Verifier chooses tags from the selected CoRIMs - including CoMID, CoSWID, CoBOM, and CoTS.
+The Verifier chooses tags from the selected CoRIMs—including CoMID, CoSWID, CoBOM, and CoTS.
 
 The Verifier MUST discard all tags which are not syntactically and semantically valid.
-In particular, any cross-referenced triples (e.g., CoMID-CoSWID linking triples) MUST be successfully resolved.
+In particular, any cross-referenced triples (e.g., CoMID-CoSWID linking triples and CoBOM-listed tags) MUST be successfully resolved.
 
-#### CoBOM Extraction
+#### Tag activation by CoBOM
 
-This section is not applicable if the Verifier appraisal policy does not require CoBOMs.
+This step can be skipped of all tags are implicitly active.
 
 CoBOMs which are not within their validity period are discarded.
-
-The Verifier processes all CoBOMs that are valid at the point in time of Evidence Appraisal and activates all tags referenced therein.
+The Verifier processes all CoBOMs that are valid at the point in time of Evidence Appraisal.
 
 A Verifier MAY decide to discard some of the available and valid CoBOMs depending on any locally configured authorization policies.
 (Such policies model the trust relationships between the Verifier Owner and the relevant suppliers, and are out of the scope of the present document.)
 For example, a composite device ({{Section 3.3 of -rats-arch}}) is likely to be fully described by multiple CoRIMs, each signed by a different supplier.
 In such case, the Verifier Owner may instruct the Verifier to discard tags activated by supplier CoBOMs that are not also activated by the trusted integrator.
 
-After the Verifier has processed all CoBOMs it MUST discard any tags which have not been activated by a CoBOM.
-
-### Evidence Collection {#sec-ev-coll}
-
-During the Evidence collection phase, the Verifier communicates with Attesters to gather Evidence.
-The first part of this phase does not require any cryptographic validation.
-This means that Verifiers can use untrusted code to discover Evidence sources.
-Attesters are Evidence sources.
-
-Verifiers may rely on conveyance protocol specific context to identify an Evidence source, which is the Evidence input oracle for appraisal.
-
-The collected Evidence is then transformed to an internal representation, making it suitable for appraisal processing.
+If the Verifier Policy requires CoBOMs, then after the Verifier has processed all CoBOMs it MUST discard any tags which have not been activated by a CoBOM.
```

Comment:
Removal of sec-ev-coll goes too far. This section corresponds to the CoRIM Selection, only for Evidence. I suggest:
```
#### Evidence Selection

All available Evidence in the Appraisal Session's inputs are checked for validity.

Evidence that is not within it's validity period, or that cannot be associated with an authenticated and authorized source MUST be discarded.

Evidence that has been secured by a cryptographic mechanism, such as a signature, that fails validation MUST be discarded.

Evidence selection MUST yield at least one usable entry.

Selected Evidence is transformed into an internal representation (see {{sec-phase1-trans}}).
```
The new text doesn't seem to belong in this section.

Reply:
Added.